resources/assets/less/bem/profile-links.less

```diff
@@ -36,4 +42,27 @@
     font-weight: 600;
     word-wrap: break-word;
   }
+
+  .edit-profile-links {
+    position: absolute;
+    top: 0;
+    right: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 70px;
+    height: 100%;
+
+    a {
+      color: @osu-colour-c1;
+      button {
+        background-color: @osu-colour-b2;
+        outline: none;
+        border: none;
+        border-radius: 50%;
+        width: 40px;
+        height: 40px;
+      }
+    }
```

Comment:
just use `profile-page-toggle`. The styling will need a bit of update so it also works with `a` element.